Lombiq.Tests.UI/SecurityScanning/SecurityScanningUITestContextExtensions.cs

```diff
@@ -25,7 +27,14 @@ public static class SecurityScanningUITestContextExtensions
         Action<SarifLog> assertSecurityScanResult = null) =>
         context.RunAndAssertSecurityScanAsync(
             AutomationFrameworkPlanPaths.BaselinePlanPath,
-            configure,
+            configuration =>
+            {
+                // Make sure to visit at least one page that throws an exception.
+                var errorPageRelativeUrl = context.GetRelativeUrlOfAction<ErrorController>(error => error.Index());
+                configuration.ModifyZapPlan(plan => plan.AddUrl(context.GetAbsoluteUri(errorPageRelativeUrl)));
```

Comment:
Why? I figure we always want to do the baseline scan so it's enough to verify this once.

Reply:
The Baseline Full scans are mutually exclusive since Full also runs what the Baseline does. The two API scans are unique but you wouldn't necessarily run Baseline or Full for the same app if it's just a bunch of APIs.

Reply:
Doesn't matter now, since I've removed it form here as well.

Reply:
Why do? It's useful to check an error page, that's a good addition, just it's needed for all scans.

Reply:
Always including the error page means that you _have to_ disable app log assertion (which you didn't want),  because the error page will guaranteed result in an error log. Also we enforce "OrchardCore.Diagnostics" so it's kinda redundant.

Reply:
You don't have to disable error log assertions, but you need to change its rule to exclude such expected test errors. But OK.

Reply:
It's only expected to happen if manually add the URL there.

Reply:
Let's take a step back: The point of using the Diagnostics module is to have a global error handler that prevents showing detailed error pages (when not in the Development environment, since [it allows them then](https://github.com/OrchardCMS/OrchardCore/blob/29d6ba2aa1e5c2aaff9133968391f9bc110faeac/src/OrchardCore.Modules/OrchardCore.Diagnostics/DiagnosticsStartupFilter.cs#L25)), and that allows the easy theming of "friendly" error pages. The goal of adding `ErrorController` here was to check if for an 5xx error you indeed don't get a detailed error page. Do I understand correctly?

Since while recommended, we can't assume everyone will use our security defaults, such a check seems useful. However, preventing logging would need a different default log assertion, a log filter (https://github.com/Lombiq/UI-Testing-Toolbox/issues/206) or an action that just returns an HTTP 5xxx (but that's not the same as an unhandled exception). And the Diagnostics module wouldn't work anyway (unless we run the app with a Test environment config or something). This is too much, so OK to not have it.

Reply:
In the end I made a second call to the security scan a second time with modified values. This second scan has no recursion (so it shouldn't start wandering around) and the app log assertions are disabled just for its duration.

Reply:
Great!